packages/engine/arrow-js/package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "arrow",
+  "version": "1.0.0",
+  "private": "true",
+  "description": "",
+  "keywords": [],
+  "license": "MIT",
+  "author": "",
+  "type": "module",
+  "scripts": {
+    "bundle:arrow": "webpack"
+  },
+  "dependencies": {
+    "@zxing/text-encoding": "^0.9.0",
+    "apache-arrow": "^7.0.0"
+  },
+  "devDependencies": {
+    "html-webpack-plugin": "^5.5.0",
+    "webpack": "^5.69.0",
+    "webpack-cli": "^4.9.2"
```

Comment:
If configuring webpack ends up too painful, you can also try https://parceljs.org or some other zero-config bundle.